frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -409,14 +411,26 @@ export class HtmlDocument extends React.Component {
   handleSaveHtmlDocument = async () => {
     const { state, props } = this
 
+    let newDocumentForApiWithMention
+    try {
+      newDocumentForApiWithMention = wrapMentionInSpanTag(state.content.raw_content)
+    } catch (e) {
+      if (e === EXCEPTION_MENTION_PARSING) {
+        this.sendGlobalFlashMessage(props.t('Error while detecting the mentions'))
+      } else {
+        this.sendGlobalFlashMessage(props.t('Error while saving new version'))
```

Comment:
```suggestion
        this.sendGlobalFlashMessage(props.t('Error while saving the new version'))
```
(and translation strings)

Reply:
Changed